Application.hs

```diff
@@ -162,22 +165,18 @@ develMain = develMainHelper getApplicationDev
 -- | The @main@ function for an executable running this site.
 appMain :: IO ()
 appMain = do
-    -- Get the settings from all relevant sources
-    settings <- withEnv $ loadAppSettingsArgs
-        -- fall back to compile-time values, set to [] to require values at runtime
-        [configSettingsYmlValue]
-
-        -- allow environment variables to override
-        useEnv
-
-    -- Generate the foundation from the settings
+    settings <- getAppSettings
```

Comment:
I don't think a separate executable solves the issue, because we'll still need to load the app settings in the other executable. Loading the settings differently for different executables seems confusing to me.